pyproject.toml

```diff
@@ -41,15 +41,14 @@ Changelog = "https://pypdf.readthedocs.io/en/latest/meta/CHANGELOG.html"
 
 [project.optional-dependencies]
 full = [
-    "cryptography; python_version >= '3.7'",
```

Comment:
Please do these cleanups in a separate PR.

Reply:
the current code of pypdf is no more compatible with python 3.7.
I've just remove any reference to this release to prevent mistakes

Reply:
We did not do this before, although we might have introduced incompatibilities. Apart from this, this is still unrelated to the release PR. If you want to change this, please propose this in a separate PR.

Reply:
I personnally see no advantage to handle that in a separate PR. as I've asked you to manage this clean up whilst managing the crypto section